From 6d0ff72f854f85a1590d5c9337b2334fff5a1528 Mon Sep 17 00:00:00 2001
Message-Id: <6d0ff72f854f85a1590d5c9337b2334fff5a1528.1369658547.git.minovotn@redhat.com>
In-Reply-To: <07146f8b79923c529fd93fa528e6fcbd6f571a02.1369658547.git.minovotn@redhat.com>
References: <07146f8b79923c529fd93fa528e6fcbd6f571a02.1369658547.git.minovotn@redhat.com>
From: Fam Zheng <famz@redhat.com>
Date: Mon, 20 May 2013 03:36:43 +0200
Subject: [PATCH 28/47] vmdk: remove unneeded variable assignment

RH-Author: Fam Zheng <famz@redhat.com>
Message-id: <1369021022-22728-29-git-send-email-famz@redhat.com>
Patchwork-id: 51464
O-Subject: [PATCH RHEL-6.5 qemu-kvm v3 28/47] vmdk: remove unneeded variable assignment
Bugzilla: 960685
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>

From: Pavel Borzenkov <pavel.borzenkov@gmail.com>

Spotted by Clang Analyzer

Signed-off-by: Pavel Borzenkov <pavel.borzenkov@gmail.com>
Signed-off-by: Stefan Hajnoczi <stefanha@linux.vnet.ibm.com>
(cherry picked from commit c18a1c88059bd546a198d95a10e9e17b11d062d1)

Signed-off-by: Fam Zheng <famz@redhat.com>
---
 block/vmdk.c | 1 -
 1 file changed, 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 block/vmdk.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/block/vmdk.c b/block/vmdk.c
index ab8206c..757363b 100644
--- a/block/vmdk.c
+++ b/block/vmdk.c
@@ -1409,7 +1409,6 @@ static int vmdk_create(const char *filename, QEMUOptionParameter *options)
             bdrv_delete(bs);
             return -EINVAL;
         }
-        filesize = bdrv_getlength(bs);
         parent_cid = vmdk_read_cid(bs, 0);
         bdrv_delete(bs);
         relative_path(parent_filename, sizeof(parent_filename),
-- 
1.7.11.7

